### PR TITLE
change internal variable to proper + fix typos and small PEP8 issues

### PR DESCRIPTION
**Pull request by Rogalek**

### CODE REVIEW

File: axie-scholar-utilities/source/axie/breeding.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def calculate_breeding_cost(self):
 
     def calculate_fee_cost(self):
         number_of_breeds = len(self.breeding_file)
+        cost = 0
```
Comment (FerranMarin) [inline axie-scholar-utilities/source/axie/breeding.py:141]: Redudant?

File: axie-scholar-utilities/source/axie/payments.py [hunk 2]
```diff
@@ -240,8 +239,8 @@ def verify_inputs(self):
             self.type = "new"
         except ValidationError as ex:
             new_msg = ("If you were tyring to pay using the current format:\n"
-                          f"Error given: {ex.message}\n"
-                          f"For attribute in: {list(ex.path)}\n")
+                       "Error given: {ex.message}\n"
```
Comment (FerranMarin) [inline axie-scholar-utilities/source/axie/payments.py:242]: forgot the f', this would format it wrong

**Comment by FerranMarin** [comment]
Few minor comments, please fix for this PR to be merged :)

File: axie-scholar-utilities/source/axie/breeding.py (hunk 1 above)
Comment (Rogalek) [reply axie-scholar-utilities/source/axie/breeding.py:141]: if redundant I propose this use elif and else, will be in new commit

File: axie-scholar-utilities/source/axie/payments.py (hunk 2 above)
Comment (Rogalek) [reply axie-scholar-utilities/source/axie/payments.py:242]: yes, you are right, fixing now

**Comment by Rogalek** [comment]
fix added + refactor for `calculate_fee_cost`

File: axie-scholar-utilities/source/axie/breeding.py (hunk 1 above)
Comment (FerranMarin) [reply axie-scholar-utilities/source/axie/breeding.py]: fair enough

**Pull request merged**